pkg/controller/job/metrics/metrics.go

```diff
@@ -80,9 +80,10 @@ var (
 	//   result:          failed, succeeded
 	JobPodsFinished = metrics.NewCounterVec(
 		&metrics.CounterOpts{
-			Subsystem: JobControllerSubsystem,
-			Name:      "job_pods_finished_total",
-			Help:      "The number of finished Pods that are fully tracked",
+			Subsystem:      JobControllerSubsystem,
+			Name:           "jobs_pods_finished_total",
+			Help:           "The number of finished Pods that are fully tracked",
```

Comment:
nit:
```suggestion
			Name:           "pods_finished_total",
			Help:           "The number of finished Pods that are fully tracked in Job statuses",
```

/hold cancel

Reply:
oops, I meant
/lgtm cancel

Reply:
prefixing with `job` is also fine with me since it adds a bit of context, but we can also remove it, both make sense in my opinion

Reply:
In that case, we can leave it as is to avoid disturbing existing users of this metric.

Reply:
sounds good to me, I would just change to singular `pod` since it is used as a way to scope where the pods are coming from rather than as a quantitative object

Reply:
ah yes, I totally meant singular job